lib/purchases_flutter.dart

```diff
@@ -271,13 +271,15 @@ class Purchases {
     String productIdentifier, {
     UpgradeInfo? upgradeInfo,
     PurchaseType type = PurchaseType.subs,
+    bool? isPersonalizedPrice,
```

Comment:
I think we need to add this parameter to the docs of these purchase methods. Also mention that this is Android only.